src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs

```diff
@@ -599,7 +599,8 @@ private InternalBackupResult CreateLocalBackupOrSnapshot(PeriodicBackupStatus st
                     else
                     {
                         // snapshot backup
-                        ValidateFreeSpaceForSnapshot(tempBackupFilePath);
+                        var excludeIndexes = _configuration.SnapshotSettings?.ExcludeIndexes ?? false;
```

Comment:
Can be inside `ValidateFreeSpaceForSnapshot`

Reply:
Done.